webapp/channels/src/i18n/en.json

```
@@ -5633,6 +5633,8 @@
   "user.settings.display.oneClickReactionsOnPostsDescription": "When enabled, you can react quickly with recently used reactions when hovering over a message.",
   "user.settings.display.oneClickReactionsOnPostsTitle": "Quick reactions on messages",
   "user.settings.display.preferTime": "Select how you prefer time displayed.",
+  "user.settings.display.renderEmoticonsAsEmojiDesc": "When enabled, text emoticons in messages will automatically be rendered as emojis (For example :D as 😄)",
+  "user.settings.display.renderEmoticonsAsEmojiTitle": "Auto-render emoticons as emojis",
```

This needs to be updated to match the new string in the file. Unfortunately, the tooling that extracts these from the code doesn't handle when there's a mismatch between the two
```suggestion
  "user.settings.display.renderEmoticonsAsEmojiTitle": "Render emoticons as emojis",
```